src/main/resources/config/liquibase/changelog/20211901120000_changelog.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <!-- More information about the changes in this changelog can be found in the associated PR https://github.com/ls1intum/Artemis/pull/2651 -->
+
+    <!-- Adding a new enum to the exercise table that signifies if and how this exercise counts towards to the course / exam score -->
+    <changeSet author="swaldhauser" id="20211901120000-1">
```

Comment:
please merge all changes into one changeset

Reply:
As discussed on Slack we will leave the different changeSets so that the order of execution is fixed.